iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/helpers/IntegrationTest.java

```diff
@@ -77,6 +77,9 @@ protected void finished(Description description)
     public static boolean runProvisioningTests = Boolean.parseBoolean(Tools.retrieveEnvironmentVariableValue("RUN_PROVISIONING_TESTS", "true"));
     public static boolean runDigitalTwinTests = Boolean.parseBoolean(Tools.retrieveEnvironmentVariableValue("RUN_DIGITAL_TESTS", "true"));
 
+    // Determines if the tear down for a given test should delete the device identity, or recycle it so that another test can use it
+    public static boolean recycleIdentities = Boolean.parseBoolean(Tools.retrieveEnvironmentVariableValue("RECYCLE_TEST_IDENTITIES", "false"));
```

Comment:
The "false" here is just the default value. I figure people running test locally would rather the test delete the identity for them